src/app/components/PriceChart.tsx

```diff
@@ -37,30 +58,45 @@ function PriceChartCanvas(props: PriceChartProps) {
       // OHLC
       const ohlcSeries = chart.addCandlestickSeries({});
       ohlcSeries.setData(clonedData);
+
       chart.priceScale("right").applyOptions({
+        //MODIFY THEME COLOR HERE
+        borderColor: "#d3d3d4",
         scaleMargins: {
-          top: 0,
-          bottom: 0.2,
+          top: 0.1,
+          bottom: 0.3,
         },
       });
 
-      // Volume
+      // Volume Initialization
       const volumeSeries = chart.addHistogramSeries({
         priceFormat: {
           type: "volume",
         },
         priceScaleId: "volume",
-        color: "#eaeff5",
       });
 
-      volumeSeries.setData(clonedData);
+      // VOLUME BARS
+      // MODIFY THEME COLOR HERE
+      volumeSeries.setData(
+        data.map((datum) => ({
+          ...datum,
+          color: datum.close - datum.open <= 0 ? "#ef5350" : "#26a69a",
```

Comment:
These colors should come from css variables. As strings in code they can be represented as `hsl(var(--su))` for green, `hsl(var(--er))` for red.

Reply:
When I tried to use the hsl(var(--su)) variables it cause the chart to be black (randomly hit or miss), I suspect it was because the library does some extra steps with the string
![image](https://github.com/DeXter-on-Radix/website/assets/3978285/6640939d-5910-450c-a3bf-ac0e804bd0e8)

Reply:
ok, probably we can import tailwind config object then and take the color from there.